.github/actions/update-mod-version/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Update module version'
+description: 'Update plugin module version in target repo.'
+inputs:
+  branch:  # id of input
+    description: 'Branch of target repo to checkout.'
+    required: true
+    default: 'master'
+  update-strategy:  # id of input
+    description: 'How to update the go module.'
+    required: true
+    default: '@latest'
+  working-directory:
```

Comment:
This is required because the bash script assumes it starts in the repo to update right? Given the script needs write permission, it will presumably have read permission, so you could get rid of the configuration, and have the script do the clone itself - that would simplify using the action a bit.

Alternatively, I would say get rid of the default, because in its current state, it definitely is a required input.

Reply:
Thanks this is a great idea! I had wanted to include the target repo checkout action as part of my action (as it makes sense to be part of the action flow) but realised you couldn't use an action within an action. Totally didn't think of just doing it in the script 🤦‍♀️ . Have added this to the script now and removed the redundant checkout action from the workflows. It also simplifies things a bit when getting the target repo name in the script.